app/views/govuk_wrapper.scala.html

```diff
@@ -74,11 +74,18 @@
         @if(contentHeader.isDefined) {
             @uiLayouts.main_content_header(contentHeader = contentHeader.get)
         }
-        @if(!isBackLinkDisable) {
-            <div class="js-visible">
-                <p><a id="back-link" class="link-back" href="javascript:history.go(-1)">@messages("charities_login.button.back")</a></p>
-            </div>
-        }
+
```

Comment:
The HMRC logo doesn't get converted into welsh, all you need to do it - https://github.com/hmrc/cgt-calculator-resident-properties-frontend/blob/master/app/views/govuk_wrapper.scala.html#L97

Reply:
> It might be unrelated to your changes, but where is the 'Is there anything wrong with this page?' OR 'Get help with this page' at the bottom of the page?

Had a discussion with Helen regarding content, It should be "Get help with this page" and Blaine will create a story for this? I know this is a very minor change :)